src/status_im/signing/core.cljs

```diff
@@ -151,35 +151,10 @@
-          (cond-> {:db (update db :signing/sign assoc :error nil :in-progress? true)}
-            (nil? action)                                 (assoc :signing/send-transaction-fx
-                                                                 {:tx-obj          tx-obj-to-send
-                                                                  :hashed-password hashed-password
-                                                                  :cb              cb})
-            (= action constants/ens-action-type-register) (assoc :json-rpc/call
-                                                                 [{:method "ens_register"
-                                                                   :params [chain-id tx-obj-to-send
-                                                                            hashed-password username
-                                                                            public-key]
-                                                                   :on-success
-                                                                   #(do
-                                                                      (cb (types/clj->json
-                                                                           {:result %}))
-                                                                      (watch-ens-tx-fn %))
-                                                                   :on-error #(cb (types/clj->json
-                                                                                   {:error %}))}])
-            (= action
-               constants/ens-action-type-set-pub-key)     (assoc :json-rpc/call
-                                                                 [{:method     "ens_setPubKey"
-                                                                   :params     [chain-id tx-obj-to-send
-                                                                                hashed-password username
-                                                                                public-key]
-                                                                   :on-success #(do (cb (types/clj->json
-                                                                                         {:result %}))
-                                                                                    (watch-ens-tx-fn %))
-                                                                   :on-error   #(cb (types/clj->json
-                                                                                     {:error
-                                                                                      %}))}])))))))
+          {:db                          (update db :signing/sign assoc :error nil :in-progress? true)
+           :signing/send-transaction-fx {:tx-obj          tx-obj-to-send
+                                         :hashed-password hashed-password
+                                         :cb              cb}})))))
```

Comment:
we don't take care of `watch-ens-tx-fn` anymore?

Reply:
in my understanding this code is outdated, so watch-ens-tx-fn is not used, but probably it was missed during refactoring, i'll take a look

Reply:
it's outdated, but it's still work i think, further refactoring is needed if we want align with latest implementation in status-go

Reply:
thanks i'll take a look deeper then